asset-bundle-converter/Assets/AssetBundleConverter/LODsConverter/Utils/WebRequestManager.cs

```diff
@@ -1,23 +1,43 @@
-﻿using System.IO;
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
 using System.Threading.Tasks;
 using Cysharp.Threading.Tasks;
+using Unity.Plastic.Newtonsoft.Json;
 using UnityEngine;
 using UnityEngine.Networking;
 
 namespace AssetBundleConverter.LODsConverter.Utils
 {
-    public class URLFileDownloader : IFileDownloader
+    public class WebRequestManager : IWebRequestManager
     {
-        private readonly string[] lodsURL;
-        private readonly string tempDownloadPath;
-
-        public URLFileDownloader(string[] lodsURL, string tempDownloadPath)
+        public async Task<Parcel> GetParcel(string sceneIDWithLODLevel)
         {
-            this.lodsURL = lodsURL;
-            this.tempDownloadPath = tempDownloadPath;
+            var decodedParcels = new List<Vector2Int>();
+
+            string hash = sceneIDWithLODLevel.Split('_')[0];
+            string url = "https://peer.decentraland.org/content/entities/active/";
+
+            using (var request = UnityWebRequest.Post(url, "{\"ids\":[\"" + hash + "\"]}", "application/json"))
+            {
+                await request.SendWebRequest();
+                if (request.result == UnityWebRequest.Result.Success)
+                {
+                    string responseText = request.downloadHandler.text;
+                    var parcelData = JsonConvert.DeserializeObject<Parcel[]>(responseText);
+                    return parcelData[0];
+                    
+                }
+                else
+                {
+                    Debug.LogError($"Error getting decoded parcels for {hash}");
+                    DCL.ABConverter.Utils.Exit(1);
+                    return null;
+                }
```

Comment:
I would put this not only after the else but also with a try-catch when doing the web request since from time to time, it fails. 

We should also cover the web request with retries